package.json

```diff
@@ -1,65 +1,66 @@
 {
-	"name": "rspell",
-	"version": "1.6.2",
```

Comment:
`.editorconfig` doesn't cover `.json` files 😢

Reply:
Ahh! Sorry about that! I'll try to fix that when I get home.